### ISSUE TITLE
Data Prepration(T1)

### ISSUE THREAD

**Issue by shantipriyap** [OdiaGenAI/OdiaGenAI_Interns_2023#1]
1. **T1_ST1** - Translating instruction set from English - XX using the existing translation app available in the OdiaGenAI Github.
2. **T1_ST3** - Validation of instruction set (please refer to the https://drive.google.com/drive/u/0/folders/16GUxTEbcvE-RL-JNrTOtGliaPBDZawso).
3. **T1_ST4** - Generate instructions manually in Odia considering the local context in mind.
4. **T1_ST6** - Prepare an evaluation test set in Odia/Indic languages for evaluating pre-trained model performance.
